contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseRecordReader.java

```diff
@@ -97,6 +97,7 @@ public HBaseRecordReader(Connection connection, HBaseSubScan.HBaseSubScanSpec su
   @Override
   protected Collection<SchemaPath> transformColumns(Collection<SchemaPath> columns) {
     Set<SchemaPath> transformed = Sets.newLinkedHashSet();
+    Set<String> completeFamilies = Sets.newHashSet();
```

Comment:
Do we need to worry about case sensitivity here?

Reply:
I observed that the planner takes care of it. It returns a single column family if there are more than one column family with same name but different case.
I still made the change to make it case insensitive.